Discord Bot/bot.js

```diff
@@ -13,6 +13,7 @@ const client = new Discord.Client({
 	messageCacheMaxSize: 50,
 	messageCacheLifetime: 7300, //Maximum poll time = 7200, ensure message not swept.
 	messageSweepInterval: 600,
+	allowedMentions: { parse: ['users'] }, // allowedMentions to prevent unintended role and everyone pings
```

Comment:
Checked through database and most custom announcement/poll messages contain some sort of role. Don't believe we should limit the bot from being able to mention them.

Reply:
Sure, can allow any mentions when sending the poll message